### PR TITLE
Scrutinizer Auto-Fixes

### PR DESCRIPTION
**Pull request by scrutinizer-auto-fixer**
@MorrisJobke requested this pull request.

It consists of patches automatically generated for this project on Scrutinizer:
https://scrutinizer-ci.com/g/owncloud/core/

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@owncloud-bot this is okay to test

**Comment by DeepDiver1975** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
:+1: from my side

**Comment by MorrisJobke** [comment]
@nickvergessen @rullzer Mind to review?

File: apps/files_external/lib/smb.php [hunk 1]
```diff
@@ -33,7 +33,6 @@
 use Icewind\SMB\Exception\NotFoundException;
 use Icewind\SMB\NativeServer;
 use Icewind\SMB\Server;
-use Icewind\Streams\CallbackWrapper;
```
Comment (nickvergessen) [inline apps/files_external/lib/smb.php]: :-1:

Comment (nickvergessen) [inline apps/files_external/lib/smb.php]: bildschirmfoto vom 2015-05-19 15 12 47

Comment (nickvergessen) [inline apps/files_external/lib/smb.php]: Seems like the wrong camelcase is throwing scrutinizer off

**Comment by nickvergessen** [comment]
Other files are okay

File: apps/files_external/lib/smb.php (hunk 1 above)
Comment (MorrisJobke) [inline apps/files_external/lib/smb.php]: Thanks. I also checked all occurences by hand, but didn't noticed that one.

**Comment by nickvergessen** [comment]
:+1:

**Comment by nickvergessen** [comment]
@DeepDiver1975 jenkins requires it's master for confirmation :shipit:

**Comment by LukasReschke** [comment]
@owncloud-bot Retest this please.

**Comment by bantu** [comment]
Hum. I really don't like that there is neither a description of what is being done in the PR nor the commit message. I assume this is an attempt to remove unused use statements.

**Pull request merged**